### DISCUSSION TITLE
Crop destruction

### DISCUSSION THREAD

**Discussion by ConstructionGamer** [Courseplay/courseplay#7504] [Q&A] [answered]
Is there a way to enable crop destruction for harvesters when working a Course? Is there an option I've missed?

The Varitron potato harvester I'm currently using doesn't destroy any crop when driving over it. Suppose it's good in a way, but prefer crop destruction on.

**Comment by Tensuko** [comment, marked as answer]
Not as option, just in the code.

**Reply by rohne83** [reply]
Can it be changed just for CP at all? Normal helper and AD also don't destroy the crop when running through it, no matter what's set in overall game settings. So might be even caused by Giants' code?

**Reply by Tensuko** [reply]
Not sure, we never tried it out.
But since CP and AD just hire normal AI Driver, it might be the case...

**Reply by crcarmichael** [reply]
> 

How is crop destruction enabled in code?
